grails-app/views/wps/jobComplete.gsp

```diff
@@ -0,0 +1,11 @@
+Hi,
+
+Your IMOS download is ready. You can download it by clicking here: ${jobReportUrl}
```

Comment:
@danfruehauf , the text is pretty much OK.  Perhaps change slightly to read:
Your IMOS subset has been created. You can download it by clicking here: